src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="Homework18"/>
+            <method name="playSong"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/229086893-ca3f166c-4938-499f-b419-e842e1087cf4.png)

![image](https://user-images.githubusercontent.com/104761482/229087987-018aa374-7616-4482-9638-8ebba8fc04c2.png)

HINT

Reply:
Hi, i fixed it and created a new pull request.